sizzle/utils/__init__.py

```
@@ -0,0 +1,27 @@
+# Make utils a package
+import requests
+
+
+def get_github_issue_title(github_issue_url):
+    """Helper function to fetch the title of a GitHub issue."""
+    try:
+        repo_path = "/".join(github_issue_url.split("/")[3:5])
+        issue_number = github_issue_url.split("/")[-1]
+        github_api_url = f"https://api.github.com/repos/{repo_path}/issues/{issue_number}"
+        response = requests.get(github_api_url)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)